### DISCUSSION TITLE
关于https转发的问题

### DISCUSSION THREAD

**Discussion by mao13820** [natfrp/wiki#67] [使用问题] [answered]
经过测试，发现腾讯香港轻量节点1存在一个问题。就是用两个域名（例如A和B）开了两个不同本地端口（例如对应443和1443）的HTTPS转发。

之后在短时间内用HTTPS访问域名A，再访问域名B，会出现访问域名B时，本应转发给1443的数据被转发给了443，也就是A域名的本地端口，反之亦然。

请帮忙查看一下这个问题，谢谢。

**Comment by fakeboboliu** [comment, marked as answer]
你好，这是符合预期的行为，浏览器对 tls session 做出了复用

解决方案可以为：
 - 使用粒度更小的证书，两个域名证书不同即不会被复用
 - 多个隧道使用不同的隧道节点
 - 使用统一后端以合并隧道

**Reply by mao13820** [reply]
> 你好，这是符合预期的行为，浏览器对 tls session 做出了复用
> 
> 解决方案可以为：
> 
> * 使用粒度更小的证书，两个域名证书不同即不会被复用
> * 多个隧道使用不同的隧道节点
> * 使用统一后端以合并隧道

非常感谢你的回答，我将按照你的建议处理。

同时还想询问一下，国内的节点是会屏蔽海外IP的访问吗？我在帮助文档中似乎没有看到，但实际中我开了台州1和成都5线路的TCP映射，被映射的设备在国内，然后访问者在国外，发现国外无法访问。不知是否有可以海外访问的国内节点，如何判断，谢谢。

**Reply by fakeboboliu** [reply]
> 国内的节点是会屏蔽海外IP的访问吗？

并没有，至少现在没有
